docs/source/common_issues.rst

```diff
@@ -808,3 +808,78 @@ not necessary:
     class NarrowerArgument(A):
         def test(self, t: List[int]) -> Sequence[str]:  # type: ignore[override]
             ...
+
+TypeVars are not the same
+------------------------------------------------
+
+As discussed in the issue
+`TypeVars should be allowed to be the same <https://github.com/python/mypy/issues/7864>`_,
```

Comment:
Style nit: we don't add links to mypy issues in the documentation, since the discussions in issues may be too technical for many readers (further information is likely needed by beginners). In this case it's okay to not have any link at all, or you can try to find a source which is more geared towards general developer population.